web/models.py

```
@@ -1,26 +1,38 @@
+import hashlib
+import logging
+import math
 import os
 import random
 import string
 import time
 import uuid
+from datetime import date, timedelta
+from decimal import Decimal
 from io import BytesIO
+from typing import Any, ClassVar, Dict, List, Optional, Tuple, Union
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)